src/cli/common.rs

```diff
@@ -5,7 +5,7 @@ use std::fmt::Display;
 use std::fs;
 use std::io::{BufRead, ErrorKind, Write};
 use std::path::Path;
-use std::sync::Arc;
+use std::rc::Rc;
```

Comment:
I think this one will get resolved as we move further into async. I think we should just permit it or ignore it if its not stabilised.

Reply:
Updated. Make sense.